src/kOS/Utilities/VesselUtils.cs

```diff
@@ -318,7 +389,312 @@ public static object GetSolarPanelStatus(Vessel vessel)
 
         public static void SolarPanelCtrl(Vessel vessel, bool state)
         {
-            vessel.rootPart.SendEvent(state ? "Extend" : "Retract");
+            foreach (var p in vessel.parts)
+            {
+                foreach (var c in p.FindModulesImplementing<ModuleDeployableSolarPanel>())
+                {
+                    if (state) { c.Extend(); }else { c.Retract(); }
+                }
+            }
+        }
+
+        public static object GetRadiatorStatus(Vessel vessel)
+        {
+            var atLeastOneRadiator = false; // No radiators at all? Always return false
+
+            foreach (var p in vessel.parts)
+            {
+                foreach (var c in p.FindModulesImplementing<ModuleActiveRadiator>())
+                {
+                    atLeastOneRadiator = true;
+
+                    if (! c.IsCooling)
```

Comment:
Yeah, I mostly don't like the syntax inconsistency of it.  Functionally, it's arguable if toggle really needs to exist for anything outside of stock action groups (even then, it's essentially `set ag1 to not ag1.`).  I'll leave the fix out for now, we'll see what we get for comments in release.